### PR TITLE
Update stdlib Versions in Ballerina Distribution

### PR DESCRIPTION
**Pull request by TharindaDilshan**
Fixes: #714

### CODE REVIEW

File: release/src/version_update/main.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            moduleList = json.load(f)
+    except:
+        print('Failed to read stdlib_modules.json')
+        sys.exit()
+
+    return moduleList['modules']
+
+# Fetch ballerina-distribution repository with GitHub credentials
+def configureGithubRepository():
+    g = Github(packagePAT)
```
Comment (ThisaruGuruge) [inline release/src/version_update/main.py:41]: Shall we use a proper name for this?
Please check the other places as well.

File: release/src/version_update/main.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+    data = file.decoded_content.decode("utf-8")
+
+    return data
+
+# Update stdlib module versions in the gradle.properties file with module details fetched from stdlib_modules.json
+def updatePropertiesFile(data, modules):
+    modifiedData = ''
+    updatedModules = []
+    pointer = ''
```
Comment (ThisaruGuruge) [inline release/src/version_update/main.py:65]: What does this `pointer` do? The name is ambiguous.

Comment (TharindaDilshan) [reply release/src/version_update/main.py:65]: It points to a specific line in `gradle.properties`

Comment (ThisaruGuruge) [reply release/src/version_update/main.py:65]: Then maybe `currentLine` would be a preferable name. Shall we rename it?

Comment (TharindaDilshan) [reply release/src/version_update/main.py]: Updated

File: release/src/version_update/main.py (hunk 1 above)
Comment (ThisaruGuruge) [reply release/src/version_update/main.py:41]: I meant the variable `g` here.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**